topics/ecology/tutorials/ENA_Biodiv_submission/tutorial.md

```diff
@@ -0,0 +1,368 @@
+---
+layout: tutorial_hands_on
+
+title: Data submission using ENA upload Tool
+questions:
+- How to prepare sequences for submission to ENA?
+- How to upload raw sequences to ENA?
+objectives:
+- Manage sequencing files (AB1, FASTQ, FASTA, FASTQ.GZ)
+- Clean sequences in an automated and reproducible manner
+- Perform alignments for each sequence
+- Have the necessary sequence format to submit to ENA
+- Submit raw reads to ENA using the ENA upload Tool
+time_estimation: 2h
+key_points:
+- Quality Control on Sequences
+- Submission to ENA
+contributions:
+    authorship:
+        - Najatamk
+    testing:
+        - PaulineSGN
+        - yvanlebras
+    funding:
+        - pndb
+
+---
+
+
+This tutorial will guide you through the necessary steps to manage and prepare sequencing files (AB1, FASTQ, FASTA) for submission to the genomic database ENA.
+This workflow will take you from raw sequences in AB1 format through all the necessary steps to integrate these sequences into the ENA genomic database. We will convert the files into FASTQ and FASTA formats after performing quality control.
+Additionally, we will perform alignments with the NCBI database to ensure the accuracy of your sequences.You will then need to fill a metadata Excel template to use the ENA upload Tool.
+The worklow is made of 17 Galaxy tools, we will  present them and explain what they do.
+The goal is to present an accessible and reproductible workflow for data submission.
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Prepare raw data with the Data Submission Workflow
+
+> <hands-on-title> Data Upload </hands-on-title>
+> 1. **Create a new history** for this tutorial
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+> 2. **Import** the files.
+>
+>    ```
+>    https://data.indores.fr/api/access/datafile/3673
+>    https://data.indores.fr/api/access/datafile/3609
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+> 3. **Rename** {% icon galaxy-pencil %} your datafiles
+>    - `3673` becomes `A2_RC_8F2_B.pl_HCOI.ab1`
+>    - `3609` becomes `A12_RC_9G4_B.md_HCOI.ab1`
+>
+>    {% snippet faqs/galaxy/datasets_rename.md %}
+>
+> 4. **Check the datatype**
+>    - Make sure it is `ab1`, and change it if not.
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md %}
+>
+> 5. **Build a Collection** containing these two files
+>
+>    {% snippet faqs/galaxy/collections_build_list.md %}
+>
+{: .hands_on}
+
+
+> <hands-on-title> Run "Prepare Data Submission" Workflow </hands-on-title>
+> Workflow is accessible there:
+>
+> [https://ecology.usegalaxy.eu/u/najatamk/w/workflow-constructed-from-history-workflow-data-submission](https://ecology.usegalaxy.eu/u/najatamk/w/workflow-constructed-from-history-workflow-data-submission-1).

```

Comment:
please add this workflow to the workflows folder of the tutorial in the GTN

Reply:
Done! THANK YOU Saskia !